src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EditJudgmentCallbackHandler.java

```diff
@@ -98,14 +98,17 @@ private CallbackResponse buildConfirmation(CallbackParams callbackParams) {
     private CallbackResponse saveJudgmentDetails(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
         List<String> errors = new ArrayList<>();
+
         if (caseData.getJoIsRegisteredWithRTL() == YesOrNo.YES) {
             caseData.setJoIssuedDate(caseData.getJoOrderMadeDate());
         }
         if (caseData.getActiveJudgment() != null) {
-            caseData.setActiveJudgment(editJudgmentOnlineMapper.addUpdateActiveJudgment(caseData));
+            editJudgmentOnlineMapper.moveToHistoricJudgment(caseData, false);
```

Comment:
Why we need to move to historic judgment?

Reply:
As per Scenario 5 in acceptance criteria of CIV-14664, we need to archive the previous judgment and save the updated one as activeJudgment

Reply:
![image](https://github.com/user-attachments/assets/7169d36a-ee52-4ee4-8cbf-f1e0cb2544e2)

Reply:
It shouldnt ...this doesnt make any sense. I think its added by mistake. Why we want to move to historic judgment when judgment is edited. It should be only moved to historic when its set A side.